third_party/3/jwt/contrib/algorithms/__init__.pyi

```diff
@@ -0,0 +1,8 @@
+from typing import Callable, Any, Union
+from hashlib import _DataType
+
+# In reality, _HashAlg is a function of the type
+# that hashlib.sha256/384/512 are, but there doesn't
+# seem to be a consistent exportable name that we can reference
+# for that, so we'll just say it's a Callable here.
```

Comment:
I think you can just say `from hashlib import _Hash`, just like you are importing `_DataType` from there. In order to export it, you could write `from hashlib import _Hash` as `_HashAlg`. (In the future, once PEP 544 is in, I suppose `_Hash` would become a protocol.